docs/mint.json

```diff
@@ -70,6 +70,10 @@
     {
       "name": "All Tools",
       "url": "https://composio.dev/tools"
+    },
+    {
+      "name": "Chat with Code",
+      "url": "https://chatgpt.com/g/g-67697db23c808191a1787ea4b86ac1ce-composio"
```

Comment:
The addition of new entries in deps/docs/mint.json should be accompanied by an update to the corresponding documentation or a test to verify that the newly added URLs are valid and reachable.